tests/test_manaus/test_iam.py

```diff
@@ -0,0 +1,239 @@
+from datetime import datetime, timezone
+from unittest.mock import MagicMock
+import pytest
+
+from botocore.exceptions import ClientError
+from senza.manaus.iam import IAM, IAMServerCertificate
+
+IAM_CERT1 = {'CertificateBody': 'body',
+             'CertificateChain': 'chain',
+             'ServerCertificateMetadata': {
+                 'Arn': 'arn:aws:iam::0000:server-certificate/senza-example-com',
+                 'Expiration': datetime(2022, 6, 29, 0, 0),
+                 'Path': '/',
+                 'ServerCertificateId': 'CERTIFICATEID',
+                 'ServerCertificateName': 'senza-example-com-20150701',
+                 'UploadDate': datetime(2015, 7, 1, 16, 0, 40)}}
+
+IAM_CERT1_EXP = {'CertificateBody': 'body',
+                 'CertificateChain': 'chain',
+                 'ServerCertificateMetadata': {
+                     'Arn': 'arn:aws:iam::0000:server-certificate/senza-example-com',
+                     'Expiration': datetime(2015, 6, 29, 0, 0),
+                     'Path': '/',
+                     'ServerCertificateId': 'CERTIFICATEID',
+                     'ServerCertificateName': 'senza-example-com-20150701',
+                     'UploadDate': datetime(2015, 7, 1, 16, 0, 40)}}
+
+IAM_CERT2 = {'CertificateBody': 'body',
+             'CertificateChain': 'chain',
+             'ServerCertificateMetadata': {
+                 'Arn': 'arn:aws:iam::0001:server-certificate/senza-example-com',
```

Comment:
This must be `'Arn': 'arn:aws:iam::0001:server-certificate/senza-example-com-20150702',`